crosstool/cc_toolchain_config.bzl

```diff
@@ -2774,10 +2774,10 @@ please file an issue at https://github.com/bazelbuild/apple_support/issues/new
     ]
 
     tool_paths = {
-        "ar": "libtool",
+        "ar": ctx.file.libtool.path,
         "cpp": "/usr/bin/cpp",
         "dwp": "/usr/bin/dwp",
-        "gcc": "cc_wrapper.sh",
+        "gcc": ctx.file.cc_wrapper.path,
```

Comment:
I think there is a different way I should be doing this.

Related, I think: https://github.com/bazelbuild/bazel/issues/8438#issuecomment-1523519395

Reply:
Yeah we cover those with our current `action_config`, so I can remove these two.

Reply:
Hmm, dang:
```
(22:23:53) ERROR: /private/var/tmp/_bazel_buildkite/eb7275b476ff60c9bee15cac7036007d/external/local_config_apple_cc/BUILD:89:19: in cc_toolchain_suite rule @local_config_apple_cc//:toolchain: Tool path for 'ar' is missing
```

There is probably something else I can do though.

Reply:
Looks like some rules directly ask for `"ar"`, so we need this.